formulaire/src/main/resources/public/ts/directives/question/question-type/question-type-singleanswer-radio/question-type-singleanswer-radio.directive.ts

```diff
@@ -0,0 +1,70 @@
+import {Directive, ng} from "entcore";
+import {FormElement, FormElements, Question} from "@common/models";
+import {I18nUtils} from "@common/utils";
+import {Direction} from "@common/core/enums";
+import {RootsConst} from "../../../../core/constants/roots.const";
+
+interface IQuestionTypeSingleanswerRadioProps {
+    question: Question;
+    hasFormResponses: boolean;
+    formElements: FormElements;
+    I18n: I18nUtils;
+    Direction: typeof Direction;
```

Comment:
Par rapport à mon commentaire si on prend en considération : 

```suggestion
    question: Question;
    hasFormResponses: boolean;
    formElements: FormElements;
```